### ISSUE TITLE
Clean up testing for attach, and unattached operations

### ISSUE THREAD

**Issue by emmamei** [emmamei/cdkey#537]
There are a number of points that need to be cleaned up:
- Full unattached operations and other testing in `README.llGetAttached`
- Use of `isAttached` variable in Start and propagated elsewhere
- Use `isAttached` variable in other scripts by receiving link message
- Eliminate all possible `llGetAttached` calls

Unattached operations should have properties like:
- No spin
- No winding down
- No permissions request
- No RLV
- No dress and undress

In short, unattached keys are for a few things. For developers, an unattached key is for:
- Editing scripts without risk of data loss
- Editing notecards without risk of data loss

For users, an unattached key is for:
- Adding new poses
- Editing preferences
- Editing type notecards

The entire "attachment testing and purpose" will be documented in `README.llGetAttached`.

**Comment by emmamei** [comment]
If `isAttached` is false - what do we need to run the timer for? (`Main.lsl`)
```
                     if (lowScriptMode) llSetTimerEvent(LOW_RATE);
                else if (!isAttached)   llSetTimerEvent(60.0);
                else                    llSetTimerEvent(STD_RATE);
```

**Comment by emmamei** [comment]
Discussion: perhaps "unattached mode" should result in all operations coming to a full stop?

This would not be too hard, and would probably be best handled either in Start.lsl or in a special script in order to literally stop all scripts. Code could flag the "unattached" status so when later attached, it would reset the key to full operation.

The assumption would be that unattached mode was only for modifying the key with new poses, configurations, etc.

**Comment by emmamei** [comment]
There are `lastAttach*` variables in:
* Avatar
* Start

There are `isAttached` references in:

* Main
* Start

The variable `isAttached` is also used in the unused `cdWindDown` macro.

**Comment by emmamei** [comment]
Perhaps the "unattached mode" should try to attach itself to an avatar?

**Comment by emmamei** [comment]
All of the following were removed from the code completely:

- `isAttached`
- `lastAttach`
- `llGetAttach()`

Also removed related macros.

Next is to add new processing for dropped keys.

**Comment by emmamei** [comment]
The functions and variables mentioned are all removed...
